sale_stock_refund_picking/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+======================================
+Refund Return Pickings in Sales Orders
+======================================
+
+This module extends the functionality of sale orders to support marking some
+stock picking moves as "To refund in Sale Order" and to allow you to create
+invoices deducting the products that are returned and refunded.
+
+Usage
+=====
+
+To use this module, when some customer returns some refundable products to you
+before you created an invoice, you need to:
+
+#. Go to *Sales > Sales Orders > Create*.
+#. Choose a customer and add a product whose *Invoicing Policy* is *Delivered
+   quantities*, and input that you are selling 20 units.
+#. Confirm the sale.
+#. Go to *Delivery > Validate > Apply > Reverse*.
+#. Set *Quantity* to 5, and enable *To Refund*.
+#. Press *Return > Validate > Apply*.
+#. Return to the sale order.
+#. Press *Create Invoice > Invoiceable lines > Create and View Invoices*.
+#. The created invoice's amount is 15 units now.
```

Comment:
`The amount of the created invoice will be the difference between the sold and the returned quantity.`